### DISCUSSION TITLE
Mapping and localisation output visualisation

### DISCUSSION THREAD

**Discussion by tzavik** [stella-cv/stella_vslam#575] [Q&A]
I am trying to determine the accuracy of the map generated as well as the accuracy of the localisation when compared to physical measurements. What is the best way to get the map data out? Are there any programs i can use to visualise and measure the map data? Any advice would be greatly appreciated.

**Comment by ymd-stella** [comment]
See https://gist.github.com/ymd-stella/a60edd4242dd99ee4bcbd0747031d524
